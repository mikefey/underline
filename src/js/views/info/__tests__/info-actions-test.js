import test from 'ava';
import {
  showInfo,
  hideInfo,
} from './../info-actions';

test('showInfo action', (t) => {
  t.snapshot(showInfo());
});

test('hideInfo action', (t) => {
  t.snapshot(hideInfo());
});
