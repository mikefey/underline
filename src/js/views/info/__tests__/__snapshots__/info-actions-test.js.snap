// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hideInfo action 1`] = `
Object {
  "type": "HIDE_INFO",
}
`;

exports[`showInfo action 1`] = `
Object {
  "type": "SHOW_INFO",
}
`;
